### DISCUSSION TITLE
スリープ付きの武器を作って使うとフリーズする。

### DISCUSSION THREAD

**Discussion by Isaac45132** [ngmansion/FE8N#90] [Q&A] [answered]
たとえばトラキアのブリザードを作ったとすると、戦闘に入った瞬間フリーズします。
アニメオフだとしません。

**Comment by aera-ngmansion** [comment]
アニメオンの時だけの現象ですか…なんでしょうね？
ぱっと原因は思いつきませんが、見てみます

**Comment by aera-ngmansion** [comment, marked as answer]
再現できませんでした
![](https://i.imgur.com/7VxSd0i.gif)

（追記）以下、sigerudoさんからいただいたコメントを引用させていただきます(ツリーにぶらさがってる為、Answerに選べないので…)

https://github.com/ngmansion/FE8N/discussions/90#discussioncomment-830565

**Reply by Isaac45132** [reply]
https://www.twitch.tv/videos/1046650432
私と知り合いはこんなカンジになります。知り合いは最新８Nに更新しても
こうなるとのこと。

**Reply by aera-ngmansion** [reply]
現状では、個々人が追加で入れたパッチによるバグの疑いが強いと思ってます。

もちろん、他の発生条件が存在して私がそれを踏めてないだけとか。エミュレータ依存のバグとか。可能性は色々ですが…。ともかく自分の環境で発生させられなければ手が打てませんね。

最新のFE8Nのbpsを使用した時のバグ発生直前のセーブデータ作れません？

**Reply by sigerudo** [reply]
こんにちは。この不具合の原因が判明したので報告します
FEビルダーのパッチ集で「状態異常剣」というパッチがあり、それを当てていることが不具合の原因だそうです
最新の８Nではこのパッチを当てなくても状態異常武器が作れるみたいですが、皆その事を知らないのでこのパッチを当ててしまっているそうです
